node/podcontroller.go

```diff
@@ -109,6 +109,17 @@ type PodController struct {
 	resourceManager *manager.ResourceManager
 
 	k8sQ workqueue.RateLimitingInterface
+
+	// From the time of creation, to termination the knownPods map will contain the pods key
+	// (derived from K8's cache library) -> a *knownPod struct.
```

Comment:
It's "K8s" not "K8's" but I suggest writing Kubernetes as you do in other parts of the PR.